### DISCUSSION TITLE
Providers modules cannot be found.

### DISCUSSION THREAD

**Discussion by Denstojkov** [nextauthjs/next-auth#3374] [Help] [answered]
TS2307: Cannot find module 'next-auth/providers/credentials' or its corresponding type declarations.

Basically this line of code gives me error:
import CredentialsProvider from `next-auth/providers/credentials`

![image](https://user-images.githubusercontent.com/62388254/144859342-8584978b-5625-4d0c-b8ae-97ce4b616e4b.png)

Also any other Provider doesn't work also.

"next-auth": "^4.0.2",

Deleted node_modules installed it again same problem.

tsconfig.json,
`{
  "compilerOptions": {
    "target": "es5",
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": false,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve"
  },
  "include": [
    "next-env.d.ts",
    "**/*.ts",
    "**/*.tsx"
, "src/components/common/dropdown/useDetectOutSideClick.js"  ],
  "exclude": [
    "node_modules"
  ]
}
`

**Comment by balazsorban44** [comment]
Please upgrade `npm i next-auth@latest`

**Reply by Denstojkov** [reply]
I'm on latest version 4.0.2

Problem still persists

**Comment by jrdurant** [comment, marked as answer]
I had this happen to me today as well... Needed to change a lot of imports on version 4.0.2. 

You need this: import CredentialsProvider from 'next-auth/providers/credentials'

**Reply by Denstojkov** [reply]
For some unknown reason today it was fixed by itself

**Comment by balazsorban44** [comment]
Looks correct to me https://unpkg.com/browse/next-auth@4.0.2/providers/credentials.d.ts
